client/chat/admin/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Switch } from 'wouter'
+import { ChannelRoute } from '../route'
+import { AdminChannelView } from './channel-view'
+
+export function ChatAdmin() {
+  return (
+    <Switch>
+      <ChannelRoute
+        path='/chat/admin/channel-view/:channelId/:channelName'
```

Comment:
what is up with this path? This doesn't match any kind of pattern we have in the app

Reply:
Hmm, is `/chat/admin/:channelId/:channelName/view` better?
Assuming we'll also have `/chat/admin/:channelId/:channelName/manage`.